### PR TITLE
Properly obsolete burdock stalks

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
None

#### Purpose of change
So #68368 obsoleted burdock stalks but like... only a little bit...

#### Describe the solution
- Properly remove burdock stalk from our JSON, replaced with new plant stalks where applicable
- Reflavored most cooked burdock stalks

### CODE REVIEW

**Comment by NetSysFire** [comment]
Whoops! Thank you for fixing my blunder.

It is fine to adjust the recipes who were using burdock stalks to plant stalks. There are no differences between them, which is why I tried to merge them.

**Comment by Karol1223** [comment]
> Whoops! Thank you for fixing my blunder.
> 
> It is fine to adjust the recipes who were using burdock stalks to plant stalks. There are no differences between them, which is why I tried to merge them.

I'll probably edit the PR to do that tomorrow then. Doubt I'll have time to do so today.

**Comment by Karol1223** [comment]
A +8 -15 PR should *not* have been this confusing to work on holy shit

File: data/json/obsoletion/obsolete_items.json [hunk 1]
```diff
@@ -1163,12 +1163,5 @@
     "dispersion": 50,
     "recoil": 350,
     "effects": [ "COOKOFF", "NEVER_MISFIRES" ]
-  },
-  {
-    "type": "COMESTIBLE",
```
Comment (Maleclypse) [inline data/json/obsoletion/obsolete_items.json:1168]: Was this migrated in a different PR?

Comment (Karol1223) [reply data/json/obsoletion/obsolete_items.json:1168]: > Was this migrated in a different PR?

Yeah, migration handled in the PR linked in the description

**Pull request merged**